src/js/containers/review/ReviewDataContainer.jsx

```diff
@@ -43,6 +45,18 @@ class ReviewDataContainer extends React.Component {
     	}
     }
 
+    isGtas() {
+        ReviewHelper.isGtasWindow()
+            .then((res) => {
+                if(res.open != this.state.gtas) {
+                    this.setState({gtas: res})
```

Comment:
Since all we're passing back now is the "data" element in the JSON (which is either `null` or an object) it might make sense to set gtas to `res.data`)